src/app/services/roles.service.ts

```diff
@@ -33,6 +33,17 @@ export class RolesService {
     return [appRoles.admin, appRoles.control, appRoles.anonymous].includes(roleId);
   }
 
+  public hadEditPermissionsOnConfigItem(allowedRoles: string[] = []) {
+    try {
+      const roleId = Number(sessionStorage.getItem('roleId'));
+      const roleName = appRoles[roleId];
```

Comment:
I assume _anonymous_ will not be an issue.

Reply:
yes, that is fine. Filed will be editable for anonymous(authenticationOptional:true).